test/conftest.py

```diff
@@ -20,8 +21,19 @@ def _mock_proj_dir(env: str) -> str:
     def _mock_assets_dir(env: str) -> str:
         return os.getcwd()
 
+    def _mock_bioformats2raw() -> str:
+        """
+        Force M1 Mac to use conda x86 bioformats2raw installation (rather UGLY)
+        :return: The full path where the previously-created output files are stored
+        """
+        if platform.system() == "Linux":
+            command_loc("bioformats2raw")
+        elif platform.system() == "Darwin":
+            return "/Users/mbopf/miniconda-intel/envs/intel_base/bin/bioformats2raw"
```

Comment:
Is there no way to do a local configuration? Do you really want to hard code you path?

Reply:
Yep, you (and Philip) are right. I just got frustrated and kludged it. Found a better way to handle it with a symbolic link in my ~/bin directory.